### PR TITLE
Bump chrome version for tests

### PR DESCRIPTION
**Pull request by molly-moen**
Speculative fix for test failures

### CODE REVIEW

**Comment by sureshc** [comment]
CC @pablo-code-org who has been looking at a JIRA Task to upgrade our browser versions.

File: dashboard/test/ui/features/step_definitions/check_finish_button.rb [hunk 1]
```diff
@@ -23,7 +23,7 @@
 When /^I check that the (blockly|droplet|minecraft) free play level for "([^"]*)" shows the finish button for (small|mobile) screens/i do |level_type, level_name, screen_type|
   individual_steps <<-STEPS
     And I set up the #{level_type} free play level for "#{level_name}"
-    #{screen_type == 'small' ? 'And I change the browser window size to 1366 by 600' : ''}
+    #{screen_type == 'small' ? 'And I change the browser window size to 1366 by 636' : ''}
```
Comment (Hamms) [inline dashboard/test/ui/features/step_definitions/check_finish_button.rb:26]: just to double-check: will this be small enough for our other browsers?

**Review by Hamms — APPROVED** [review]
One question, but otherwise LGTM! (tests pending)

File: dashboard/test/ui/features/step_definitions/check_finish_button.rb (hunk 1 above)
Comment (molly-moen) [reply dashboard/test/ui/features/step_definitions/check_finish_button.rb:26]: I think so. I copied this from [Bethany's PR](https://github.com/code-dot-org/code-dot-org/pull/46232/files), and it doesn't seem to cause other tests to fail.

**Pull request merged**